src/containers/Faq/index.tsx

```diff
@@ -0,0 +1,64 @@
+import React, {FC} from 'react';
+import {HashLink} from 'components';
+
+import './Faq.scss';
+
+interface FaqContent {
+  answer: string;
+  id: string;
+  question: string;
+}
+
+const questionsAnswers: FaqContent[] = [
+  {
+    answer: 'New digital currency',
```

Comment:
now, the answer doesn't just have to be a string, but a ReactNode. So you can do something like:

`answer: <>New <A>Digital</A> currency</>,

Reply:
Done!